src/Access/ReplicatedAccessStorage.cpp

```diff
@@ -59,7 +60,7 @@ ReplicatedAccessStorage::ReplicatedAccessStorage(
     if (zookeeper_path.front() != '/')
         zookeeper_path = "/" + zookeeper_path;
 
-    initializeZookeeper();
+    initZooKeeperIfNeeded();
```

Comment:
can this be in `try-catch` block so CH doesn't fail to startup on unstable ZK connection?
(E.g. operation timeout when root nodes are being created)
All other functions will simply initialize it later on through `getZooKeeper` right?

Reply:
Failure on startup can be because of a wrong configuration and in this case it's better to stop immediately.